src/Http/RequestExecutor.ts

```diff
@@ -1181,18 +1209,18 @@ export class RequestExecutor implements IDisposable {
                 const resExceptionSchema = JsonSerializer
                     .getDefaultForCommandPayload()
                     .deserialize<ExceptionSchema>(responseJson);
-                const readException = ExceptionDispatcher.get(resExceptionSchema, response.status);
+                const readException = ExceptionDispatcher.get(resExceptionSchema, response.status, e);
                 command.failedNodes.set(chosenNode, readException);
-            } catch (_) {
-                log.warn(_, "Error parsing server error.");
+            } catch (__) {
```

Comment:
How is 2 _ better than 1 _ :) Maybe just name it `errorParsingException` or similar.

Reply:
matching java code